src/zenml/constants.py

```diff
@@ -439,6 +439,7 @@ def handle_int_env_var(var: str, default: int = 0) -> int:
 STATUS = "/status"
 STEP_CONFIGURATION = "/step-configuration"
 STEPS = "/steps"
+HEARTBEAT = "heartbeat"
```

Comment:
```suggestion
HEARTBEAT = "/heartbeat"
```
Probably, because all the other ones have it?